dashboard/app/models/game.rb

```diff
@@ -127,7 +127,7 @@ def uses_pusher?
   end
 
   def uses_small_footer?
-    app == NETSIM || app == APPLAB
+    app == NETSIM || app == APPLAB || name == "TextCompression"
```

Comment:
:+1: Sounds good.  The convention looks like it's only been applied to the `app` up until now - comparisons against `name` are always against string literals in this file.  No reason we can't generalize that though.